elegant-dark-theme.el

```diff
@@ -0,0 +1,74 @@
+;;; elegant-dark-theme.el --- A very minimal but elegant and consistent theme
+;;; Copyright (C) 2020 Nicolas P. Rougier and Nicolò Zorzetto
+;;; -------------------------------------------------------------------
+;;; Authors: Nicolas P. Rougier and Nicolò Zorzetto
+;;; -------------------------------------------------------------------
+;;; URL: https://github.com/rougier/elegant-emacs
+;;; -------------------------------------------------------------------
+;;; Version: 1
```

Comment:
I changed the version to `1`, maybe `1.0` is better? Or did you want to keep `0.1` ?

Reply:
No strong opinion on that. Maybe 1.0 if we add minor changes.